src/h5web/vis-packs/core/models.ts

```diff
@@ -63,7 +63,11 @@ export interface AxisParams {
 
 export type AxisMapping = (AxisParams | undefined)[];
 
-export type Bounds = [number, number, number];
+export interface Bounds {
+  min: number;
+  max: number;
+  positiveMin: number;
+}
```

Comment:
I changed `Bounds` into an object so that it is clear which number is the `min`, the `max` and the `positiveMin`. 

I made my visualisation crash a couple of times as I misplaced the bounds :smile: